about.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css"
+      integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY="
+      crossorigin="anonymous"
+    />
+    <title>About ANDIS</title>
+  </head>
+  <body>
+    <header>
+      <div id="secondary-nav-bar" class="container">
+        <div class="wrapper">
+          <a href="#"><i class="fa fa-mastodon"></i></a>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </div>
+      </div>
+      <div id="nav-bar">
+        <a href="./index.html" id="logo">ANDIS<span>2022</span></a>
+        <nav id="menu">
+          <span id="menu-close-btn">×</span>
+          <a href="./index.html" class="home-link">Home</a>
+          <a href="./index.html#program">Program</a>
+          <a href="./index.html#speakers">Speakers</a>
+          <a href="./index.html#sponsors">Sponsors</a>
+          <a href="#" class="active">About</a>
+        </nav>
+        <a href="#" id="menu-toggler">
+          <svg
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path stroke-width="3" d="M4 6h16M4 12h16M4 18h16"></path>
+          </svg>
+        </a>
+      </div>
+    </header>
+
+    <main>
+      <div class="container">
+        <div id="section1-about" class="wrapper">
+          <img src="./assets/img/shape-set-01.png" class="shape-set" alt="" />
+          <h1>Anambra State Town Hall Meeting 2022</h1>
```

Comment:
- [ ] Kindly there's a horizontal scrolling issue on your mobile version on the about page and it comes from this image kindly fix it to be inside your layout and don't be out of it and make scrolling.

![image](https://user-images.githubusercontent.com/35535130/169192000-18b44530-4c79-48e9-a00d-3892fe8c5b13.png)